website/src/content/docs/docs/libraries/azure-resource-manager/reference/data-types.md

```diff
@@ -223,6 +318,12 @@ model Azure.ResourceManager.ArmNoContentResponse<Message>
 | ------- | --------------------------------------------------------------------------------------- |
 | Message | The description of the response status (defaults to `Operation completed successfully`) |
 
+#### Examples
+
+```typespec
+op get(...ResourceInstanceParameters<Employee>): ArmNoContentResponse;
```

Comment:
A get operation is unlikely to return a 204 - making this a custom delete makes the most sense